learn/async/paginating_tasks.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Managing the task database — Meilisearch documentation
+description: Meilisearch uses a task queue to handle asynchronous operations. This document describes how to navigate long task queues with filters and pagination.
+---
+
+# Paginating tasks
+
+By default, Meilisearch returns a list of 20 tasks for each request when you query the [get tasks endpoint](/reference/api/tasks#get-tasks). This guide shows you how to navigate the task list using query parameters.
+
+## Configuring the number of returned tasks
+
+Use the `limit` parameter to change the number of returned tasks:
+
+<CodeSamples id="get_all_tasks_paginating_1" />
+
+Meilisearch will return a batch of tasks. Each batch of returned tasks is often called a "page" of tasks, and the size of that page is determined by `limit`:  
+
+```json
+{
+  "results": [
+    …
+  ],
+  "total": 50,
```

Comment:
Is `total`, the total number of tasks returned by 2 batches (or pages) of tasks? Or is it the absolute total number of tasks in the instance?

Reply:
`total` indicates the total number of tasks matching the query. Meaning, if my instance has 50 tasks and I request the first 20, `total` will be `50`. I'll update the text with that info.